WORKSPACE

```diff
@@ -290,6 +290,19 @@ maven_install(
     use_unsafe_shared_cache = True,
 )
 
+maven_install(
+    name = "jetify_test",
+    artifacts = [
+        "com.google.guava:guava:27.0-jre",
+        "com.android.support:appcompat-v7:28.0.0"
+    ],
+    repositories = [
+        "https://jcenter.bintray.com/",
+        "https://maven.google.com",
```

Comment:
Let's reorder and hit maven.google.com first? JCenter is not very reliable 😬

Reply:
I don't think this really matters (and it's just for tests), we'll need to hit jcenter for guava anyway.